### PR TITLE
Reset loading state when loading field statistics fails

### PR DESCRIPTION
**Pull request by edmundoa**
This PR resets the loading state when fetching statistics for a field fails.

@mehrenreich I added you to the reviewers in case you want to take a look as well :)

Refs #5836

### CODE REVIEW

File: graylog2-web-interface/src/components/field-analyzers/FieldStatistics.jsx [hunk 1]
```diff
@@ -159,28 +173,34 @@ const FieldStatistics = createReactClass({
   },
 
   _getHeaderCaret(column) {
-    if (this.state.sortBy !== column) {
+    const { sortBy, sortDescending } = this.state;
+    if (sortBy !== column) {
       return null;
     }
-    return this.state.sortDescending ? <i className="fa fa-caret-down" /> : <i className="fa fa-caret-up" />;
+    return sortDescending ? <i className="fa fa-caret-down" /> : <i className="fa fa-caret-up" />;
```
Comment (kyleknighted) [inline graylog2-web-interface/src/components/field-analyzers/FieldStatistics.jsx]: Could potentially do this a little shorter.

```suggestion
    return <i className=`fa fa-caret-${sortDescending ? 'down' : 'up'}` />;
```

**Comment by bernd** [comment]
@edmundoa Could you please fix the conflict so we can get this merged? :smile: Thank you!

**Comment by edmundoa** [comment]
Rebased, please double-check :)

**Comment by bernd** [comment]
@edmundoa Thank you! Will test and review! :+1:

**Pull request merged**